spec/index.html

```diff
@@ -1350,106 +1445,75 @@ <h3>The <code>rdf:XMLLiteral</code> Datatype</h3>
-    <a>IRI</a> <code>&lt;#chapter1&gt;</code> should be taken to
-    <a>denote</a> that same section in all RDF-bearing representations of the
-    same primary resource.</p>
-</section>
-
-<section id="section-generalized-rdf" class="informative">
-  <h2>Generalized RDF Triples, Graphs, and Datasets</h2>
+    <dl>
+      <dt id="JSON-uri">The IRI denoting this <a>datatype</a></dt>
+      <dd>is <code>http://www.w3.org/1999/02/22-rdf-syntax-ns#JSON</code>.</dd>
 
-  <p>It is sometimes convenient to loosen the requirements
-    on <a>RDF triple</a>s.  For example, the completeness
-    of the RDFS entailment rules is easier to show with a
-    generalization of RDF triples.</p>
+      <dt id="JSON-lexical-space">The <a>lexical space</a></dt>
+      <dd>is the set of all <a>strings</a> that conform to the
+        <a data-cite="RFC4627#section-2">JSON Grammar</a> as described in
+        <a data-cite="RFC8259#section-2">Section&nbsp;2 JSON Grammar</a> of
+        [[RFC8259]].</dd>
 
-  <p>A <dfn class="export">generalized RDF triple</dfn> is a triple having a subject,
-    a predicate, and object, where each can be an <a>IRI</a>, a
-    <a>blank node</a> or a
-    <a>literal</a>. A
-    <dfn class="export">generalized RDF graph</dfn>
-    is a set of generalized RDF triples. A
-    <dfn class="export">generalized RDF dataset</dfn>
-    comprises a distinguished generalized RDF graph, and zero
-    or more pairs each associating an IRI, a blank node or a literal
-    to a generalized RDF graph.</p>
+      <dt id="JSON-value-space">The <a>value space</a></dt>
+      <dd>
+        is a single <a data-cite="RFC4627#section-3">JSON value</a> in the form of an <a data-lt="list">array</a>,
+        <a>map</a>,
+        <a>string</a>,
+        <a data-cite="ECMASCRIPT#sec-terms-and-definitions-number-value">number</a>,
+        <a>boolean</a>, or
+        <a data-cite="INFRA#nulls">null</a>.
+        <ul>
+          <li><a data-lt="list">Array</a> <a data-cite="INFRA#list-item">entries</a> may be
+            <a>map</a>,
+            <a>string</a>,
+            <a data-cite="ECMASCRIPT#sec-terms-and-definitions-number-value">number</a>,
+            <a>boolean</a>, or
+            <a data-cite="INFRA#nulls">null</a>.</li>
+          <li><a>Map</a> <a data-cite="INFRA#map-key">keys</a> are <a>strings</a> with
+            <a data-cite="INFRA#map-value">values</a> may be any of the above
+            <a data-cite="RFC4627#section-3">JSON values</a>.</li>
+        </ul>
 
+        Two <a data-cite="RFC4627#section-3">JSON values</a> are considered equal
+        if they are the same if they are the same <a>string</a>,
+        <a data-cite="ECMASCRIPT#sec-terms-and-definitions-number-value">number</a>,
+        <a>boolean</a>, or
+        <a data-cite="INFRA#nulls">null</a>,
+        if they are both an <a data-lt="list">array</a> with equal <a data-cite="INFRA#list-item">entries</a>, or
```

Comment:
```suggestion
        if they are both an <a data-lt="list">array</a> with equal <a data-cite="INFRA#list-item">entries</a>
        in the same order, or
```
It's not the only way to say this e.g. saying they are pairwise equal.